### PR TITLE
Added pagerduty alerts

### PR DESCRIPTION
**Pull request by goruha**
## What
* Added `pagerduty` integration with `kube-prometheus` alerts
* Update `prometheus operator` to `0.23.2`
* Enable scrapping from `ServiceMonitors` in any namespaces

## Why
* To use `pagerduty` escalations
* To fix https://github.com/coreos/prometheus-operator/pull/1619
* Allow microservice apps to define own service monitors

### CODE REVIEW

File: helmfile.d/0400.kube-prometheus.yaml [hunk 1]
```diff
@@ -96,20 +96,27 @@ releases:
...
-            receiver: "slack_general"
+            receiver: "general"
             routes:
             - match:
                 alertname: "DeadMansSwitch"
               receiver: "null"
           receivers:
             - name: "null"
-            - name: "slack_general"
+            - name: "general"
```
Comment (osterman) [inline helmfile.d/0400.kube-prometheus.yaml]: Rename to `slack`?

Comment (osterman) [inline helmfile.d/0400.kube-prometheus.yaml:107]: Rename to `slack_configs` for consistency?

File: helmfile.d/0400.kube-prometheus.yaml [hunk 2]
```diff
@@ -226,6 +233,21 @@ releases:
...
           - port: "metrics"
             interavl: 15s
+  - prometheusRules:
+      nginx-ingress:
+        labels:
+          prometheus: kube-prometheus
+        groups:
+        - name: pageduty.rules
+          rules:
+          - alert: PagerDutyTestError
```
Comment (osterman) [inline helmfile.d/0400.kube-prometheus.yaml]: Do we want to commit this? Should it be wrapped in an ENV?

**Review by osterman — CHANGES_REQUESTED** [review]
See comments

File: helmfile.d/0400.kube-prometheus.yaml (hunk 1 above)
Comment (goruha) [reply helmfile.d/0400.kube-prometheus.yaml:107]: This is the one receiver for both - slack and pagerduty

**Review by osterman — APPROVED** [review]

**Pull request merged**